src/components/Calculator.js

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable */
+import React, { Component } from 'react';
```

Comment:
- [ ] Consider dividing the Calculator.js file in smaller components so you can practice more in passing props to child components
- [ ] Consider using lifecycle methods